source/backend/api/Controllers/LookupController.cs

```diff
@@ -76,102 +83,112 @@ public IActionResult GetPropertyClassificationTypes()
         [Produces("application/json")]
         [ProducesResponseType(typeof(IEnumerable<Model.LookupModel>), 200)]
         [SwaggerOperation(Tags = new[] { "lookup" })]
+        [ResponseCache(Duration = OneHourInSeconds)]
```

Comment:
Could use some feedback here. My worry is that a user could: 1) Log into the app 2) get a list of lookup codes 3) we do a database update 4) the list of codes is now out of date.

The risk of this in prod seem low but I can see this happening in our lower environments. Perhaps this is more trouble then its worth?

Reply:
It really depends on how often the codes get updated. AFAIK they require a migration, so not very often. I think it should be ok to cache them for an hour.